ArcGISRuntimeSDKQt_CppSamples/Maps/MapReferenceScale/README.md

```diff
@@ -0,0 +1,34 @@
+# Map reference scale
+Set the map's reference scale and which feature layers should honor the reference scale.
+
+![Map Reference Scale App](screenshot.png)
```

Comment:
Unfortunately this will not render in the sample viewer due to its limitations. Needs to be:

`![]screenshot.png)`

Goes for other README also.